openmetadata-ui/src/main/resources/ui/src/rest/ingestionPipelineAPI.ts

```diff
@@ -93,10 +93,13 @@ export const getIngestionPipelines = async (
   return response.data;
 };
 
-export const triggerIngestionPipelineById = (
-  id: string
-): Promise<AxiosResponse> => {
-  return APIClient.post(`/services/ingestionPipelines/trigger/${id}`);
+export const triggerIngestionPipelineById = async (id: string) => {
+  const response = await APIClient.post<
+    unknown,
```

Comment:
@chirag-madlani , Can you check if we can use the proper type here?

Reply:
@Sachin-chaurasiya It’s the type for data that we are sending to post method, but in this case, we aren’t sending anything hence unknown